### PR TITLE
ADEN-2556 ads-mercury-test-synthetic fix

### PR DESCRIPTION
**Pull request by nandy-andy**
Our synthetic test started to fail most probably because of browser/selenium update. We needed to update the sample file for the ad.

Additionally, I've added better logging message, so instead of:
bad-log-message

We have now clear message where to look for an issue:
better-log-message

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java [hunk 1]
```diff
@@ -321,9 +321,19 @@ public AdsBaseObject verifyLineItemId(String slotName, int lineItemId) {
...
     WebElement element = getIframe(slotName, src);
-    Assertion
-        .assertTrue(new AdsComparison().compareImageWithScreenshot(imageUrl, element, driver));
-    PageObjectLogging.log("verifyAdImage", "Ad looks good", true, driver);
+
+    try {
+      Assertion
+          .assertTrue(new AdsComparison().compareImageWithScreenshot(imageUrl, element, driver));
+      PageObjectLogging.log("verifyAdImage", "Ad looks good", true, driver);
+    } catch (AssertionError e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java]: MAJOR Either log or rethrow this exception. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by nandy-andy** [comment]
@drets share with me the wisdom since nothing comes to my mind right now :japanese_ogre:

**Comment by drets** [comment]
> Our synthetic test started to fail most probably because of browser/selenium update.

or VM update.

> @drets share with me the wisdom since nothing comes to my mind right now :japanese_ogre:

Did you try `Assertion.assertTrue(false, "HELLO WORLD");`?
:smirk_cat:

**Comment by nandy-andy** [comment]
You're right! It doesn't look that bad ;) Thanks!
better-log-message

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**